geonode/templates/index.html

```diff
@@ -14,22 +15,71 @@
   {% if block.super %}
   {{ block.super }}
   {% else %}
-  <div class="jumbotron">
-    <div class="container">
-        {% with jumbotron_welcome_title=custom_theme.jumbotron_welcome_title|default:"Welcome"|template_trans %}
-        <h1>{% trans jumbotron_welcome_title %}</h1>
-        {% endwith %}
-        <p></p>
-        {% with jumbotron_welcome_content=custom_theme.jumbotron_welcome_content|default:"GeoNode is an open source platform for sharing geospatial data and maps."|template_trans %}
-        <p>{% trans jumbotron_welcome_content %}</p>
-        {% endwith %}
-        {% if not custom_theme.jumbotron_cta_hide %}
-        <p><a class="btn btn-default btn-lg" target="_blank"
-              href="{{custom_theme.jumbotron_cta_link|default:_("https://docs.geonode.org/en/master/usage/index.html")}}"
-              role="button">{{custom_theme.jumbotron_cta_text|default:_("Get Started &raquo;")}}</a></p>
-        {% endif %}
-    </div>
+
+{% if custom_theme.welcome_theme == 'JUMBOTRON_BG' or not slides %}
+<div class="jumbotron">
+  <div class="container">
+      {% with jumbotron_welcome_title=custom_theme.jumbotron_welcome_title|default:"Welcome"|template_trans %}
+      <h1>{% trans jumbotron_welcome_title %}</h1>
+      {% endwith %}
+      <p></p>
+      {% with jumbotron_welcome_content=custom_theme.jumbotron_welcome_content|default:"GeoNode is an open source platform for sharing geospatial data and maps."|template_trans %}
+      <p>{% trans jumbotron_welcome_content %}</p>
+      {% endwith %}
+      {% if not custom_theme.jumbotron_cta_hide %}
+      <p><a class="btn btn-default btn-lg" target="_blank"
+          href="{{custom_theme.jumbotron_cta_link|default:_("https://docs.geonode.org/en/master/usage/index.html")}}"
+          role="button">{{custom_theme.jumbotron_cta_text|default:_("Get Started &raquo;")}}</a></p>
+      {% endif %}
   </div>
+</div>
+{% else %}
+<div class="geonode-slide-show">
+	<div class="jumbotron">
+	   <div class="container">
+		  <div id="myCarousel" class="carousel slide" data-ride="carousel">
+			 <!-- Indicators -->
+			 {% if slides.count > 1%}
+			 <ol class="carousel-indicators">
+				{% for slide in slides %}
+				<li data-target="#myCarousel" data-slide-to="{{ forloop.counter | add:'-1' }}" class="{% if forloop.counter == 1 %}active{%endif%}"></li>
+				{% endfor %}
+			 </ol>
+			 {% endif %}
+			 {% if custom_theme.jumbotron_welcome_title %}
+			 <h1 class="carousel-header">{{ custom_theme.jumbotron_welcome_title }}</h1>
+			 {% endif %}
+			 <!-- Wrapper for slides -->
+			 <div class="carousel-inner" role="listbox">
+				{% for slide in slides %}
+				<div class="item {% if forloop.counter == 1 %}active{%endif%}">
+					{% if slide.jumbotron_slide_image %}
+				   <img src="{{ slide.jumbotron_slide_image.url }}" alt="{{ slide.slide_name }}" style="width: 100%; height: 30em">
```

Comment:
Please avoid the use of inline style attributes. Use selectors instead.
Further I'm a bit unhappy with the slides look and feel as it get's disproportionately stretched. Please consider the use of `object-fit: cover;` or if it's making things easier the use of background images.